packages/mermaid/src/diagrams/info/infoDb.ts

```diff
@@ -13,9 +13,9 @@ const clear = (): void => {
 };
 
 const db: InfoDB = {
-  clear,
-  setInfo,
-  getInfo,
+  clear: clear,
+  setInfo: setInfo,
+  getInfo: getInfo,
 };
```

Comment:
Why do we need this?

Reply:
No, but since it's it's exists on the info grammar I left them there, I tried adding it and removing it but no effect on rendered diagram.